proxy/src/bind.rs

```diff
@@ -260,6 +273,23 @@ where
         Reconnect::new(proxy)
     }
 
+    /// Bind the _whole_ stack, including the `WatchService` that rebinds
+    /// the inner stack on TLS client configuration changes.
+    fn bind_stack(&self, ep: &Endpoint, protocol: &Protocol) -> Stack<B> {
+        debug!("bind_stack: endpoint={:?}, protocol={:?}", ep, protocol);
+        // TODO: it would be nice to not have to wrap `BindsPerRequest`
+        // services with `WatchService`...
```

Comment:
I understand the intent here, but this is a non-sequitur in this context.  `BindsPerRequest` has nothing to do with this file, as far as I can tell from this diff.  Perhaps it would be better to add a comment onto BindsPerRequest that it should be replaced with WatchService, but even this sounds doubtful to me. I'd probably leave this out.

Reply:
`BindsPerRequest` _is_ defined in this file [here](https://github.com/runconduit/conduit/pull/1185/files/ce04a004cfdbed8e9c7e7fa6e49c88988675eb20#diff-e34b50b3b8e16943508a0a2713abe6d0R79). What I meant by this comment is just that the `BindsPerRequest` services will only ever be used a single time, so it seems unnecessary to wrap them in `WatchService` as well so they could be rebound in the future, since they're _always_ rebound per request.

Reply:
Ah. _This_ is what your comment should say :D  Explain why BindsPerRequest is now redundant.  "it would be nice" is meaningless to me

Reply:
>  Explain why BindsPerRequest is now redundant

`BindsPerRequest` is still necessary, it's _putting the `WatchService` on `BindsPerRequest` services_ that's unnecessary.

> This is what your comment should say

Will do :)